app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+  before_action :set_locale
+  def set_locale
+    I18n.locale = params[:locale] || I18n.default_locale
+  end
+
+  def set_locale
+    locale = params[:locale].to_s.strip.to_sym
+    I18n.locale = I18n.available_locales.include?(locale) ?
+    locale : I18n.default_locale
```

Comment:
```suggestion
      locale : I18n.default_locale
```
Khi mà xuống dòng thì nhớ đặp 1 tab so với dòng trên nha. :D :+1: